src/model_DAforcing.py

```diff
@@ -330,12 +342,58 @@ def __init__(self, bmi_cfg_file=None):
 
             lite_restart_file = self._compute_parameters['restart_parameters']['lite_channel_restart_file']
             if lite_restart_file:
+
                 self._q0, self._t0 = _read_lite_restart(lite_restart_file)
-            
+
+                (_q0_columnArray, _q0_columnLengthArray, _q0_nCol, \
+                    _q0_indexArray, _q0_nIndex, _q0_Array) = \
+                    _bmi_disassemble_lite_restart (self._q0,np.float32)
+
+                self.q0_columnArray = _q0_columnArray
+                self.q0_columnLengthArray = _q0_columnLengthArray
+                self.q0_nCol = _q0_nCol
+                self.q0_indexArray = _q0_indexArray
+                self.q0_nIndex = _q0_nIndex
+                self.q0_Array = _q0_Array
+
             lite_restart_file = self._compute_parameters['restart_parameters']['lite_waterbody_restart_file']
             if lite_restart_file:
+
                 self._waterbody_df, _ = _read_lite_restart(lite_restart_file)
-            
+
+                (_waterbodyLR_columnArray, _waterbodyLR_columnLengthArray, _waterbodyLR_nCol, \
+                    _waterbodyLR_indexArray, _waterbodyLR_nIndex, _waterbodyLR_Array) = \
+                    _bmi_disassemble_lite_restart (self._waterbody_df,np.float64)
+
+                self.waterbodyLR_columnArray = _waterbodyLR_columnArray
+                self.waterbodyLR_columnLengthArray = _waterbodyLR_columnLengthArray
+                self.waterbodyLR_nCol = _waterbodyLR_nCol
+                self.waterbodyLR_indexArray = _waterbodyLR_indexArray
+                self.waterbodyLR_nIndex = _waterbodyLR_nIndex
+                self.waterbodyLR_Array = _waterbodyLR_Array
+
+            #testRevert = True
+
+            if (testRevert):
```

Comment:
Can we remove all the `testRevert` sections? Keep the text somewhere for later use, but we don't need it in this file anymore.